receiver/purefareceiver/config.go

```diff
@@ -57,7 +57,7 @@ type Settings struct {
 
 type ReloadIntervals struct {
 	Array       time.Duration `mapstructure:"array"`
-	Host        time.Duration `mapstructure:"host"`
+	Hosts       time.Duration `mapstructure:"host"`
```

Comment:
Why is this plural here but the config definition is singular?

Reply:
Agree. The OpenMetrics endpoint for the [PureFA OpenMetrics Exporter](https://github.com/PureStorage-OpenConnect/pure-fa-openmetrics-exporter#scraping-endpoints) is /metrics/hosts and needs to be reflected in the same plurality.